about.html

```diff
@@ -0,0 +1,94 @@
+                <li>English</li>
+                <li>my page</li>
+                <li>log in</li>
+            </ul>
+        </div>
+        <nav>
+            <div class="logo-container"><a href="index.html"><img src="assets/images/logo.png" id="logo" alt="logo image"></a></div>
+            <img src="assets/images/menu-icon.svg" id="menu-icon" alt="hamburger menu icon" class="icon">
+            <ul class="nav-links" id="nav-links">
+              <img src="assets/images/menu-close-Icon.png" id="menu-close-icon" class="icon" alt="close menu icon">
+                <li><a href="index.html">Home</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="">program</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Join</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Sponsor</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">News</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Campaign</a></li>
+                <!-- <div class="divider"></div> -->
+            </ul>
+        </nav>
+      </header>
+      <main>
+        <section class="one">
+          <span class="sub-title">Hello! Music Producers!</span>
+          <h1 class="title" id="title">Tritone Music Producers conference 2023</h1>
+          <small class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tenetur recusandae voluptatibus corporis ipsa temporibus minus incidunt deleniti in earum enim.</small>
+          <small class="contact-us">Please contact us via Email for any further questions about TMC production conference</small>
+          <a class="contact-link" href="#">Tritonemusic.co@enquiries.org</a>
+        </section>
+        <section class="two">
+            <h3>Tritone Music Co Producers Conference Logo</h2>
+            <div class="divider"></div>
+            <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Odit labore saepe dolores dolorem laboriosam pariatur eveniet aliquid? Ipsum, architecto possimus.</p>
+            <figure class="logo-img"></figure>
+        </section>
+        <section class="three">
+            <h3>See the past TMC production Conference</h3>
+            <div class="divider"></div>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore quisquam enim, atque nesciunt doloremque praesentium!</p>
+            <div class="gallery">
+                <div class="image one">
+                    <h3 class="year">2022</h3>
+                    <span class="location">TMC production conference in Accra</span>
+                </div>
+                <div class="image two">
+                    <h3 class="year">2021</h3>
+                    <span class="location">TMC production conference in Kumasi</span>
+                </div>    
+            </div>
+        </section>
```

Comment:
- [ ] I think it would be great if you add a line break here to let the user be able to know that this is a new section but in your design if anyone see that they will though it is one section but thery are more that one section so kindly add some borders here like the original design I can show you what I mean exactly 👍🏻🔥♻

- your design
![image](https://user-images.githubusercontent.com/58553711/212221943-bb3440cc-1afa-4ecf-a37f-caac987b36fd.png)

- Given design
![image](https://user-images.githubusercontent.com/58553711/212221994-cb7b252e-364c-40f8-ac5b-dd278dc1ca37.png)